packages/server/src/sync/api.ts

```diff
@@ -19,13 +20,18 @@ export class SyncApi {
     this.force = !!yn(process.env.SPINNED)
     this.schema = makeSyncRequestSchema(this.channels.list())
 
-    router.post('/sync', auth.public.auth(this.sync.bind(this)))
+    router
+      .route('/sync')
+      .post(auth.public.auth(this.sync.bind(this)))
+      .all(methodNotAllowed('POST'))
```

Comment:
If we're only going to implement methodNotAllowed for this route we might as well just not bother with it. It's not really additional security anyways.

Reply:
Again as a good practice, we should implement this everywhere. Best scenario, it narrows the attack vector on our endpoints, and worst case, we have a middleware that brings no overhead!